pkg/netceptor/conn.go

```diff
@@ -490,7 +490,7 @@ func generateServerTLSConfig() *tls.Config {
 	}
 }
 
-func verifyServerCertificate(rawCerts [][]byte, verifiedChains [][]*x509.Certificate) error {
+func verifyServerCertificate(rawCerts [][]byte, _ [][]*x509.Certificate) error {
```

Comment:
this is also an implementation of an interface, the `tls.Config` interface in crypto.go, so again, this is correct